### DISCUSSION TITLE
How can I extend the default list of values for the system field 'status'? (v9)

### DISCUSSION THREAD

**Discussion by log4shell** [directus/directus#10446] [Archive]
When creating a new collection, there's an optional system field for 'status' that by default only provides the options "draft", "published", and "archived". I know it's possible to add additional values when configuring the collection, however I would like to update the default values to extend the list with additional statuses (e.g., "review") that are applied to all collections using this field so that I don't have to make this change manually for each new collection.

Is there an easy way to do this in v9?

**Comment by rijkvanzanten** [comment]
No, there's no way to batch update fields across multiple collections

**Reply by log4shell** [reply]
Ok, yes I've seen that the values are stored in the fields table for each collection so it might be too late after a collection is already created. 

However, is there a way to change the defaults for all new collections? These values must be defined somewhere (perhaps in code or some default config?) - if you point me to the source I can probably patch my instance to use the new set of statuses for all collections. 

It would be really neat if a future release of Directus allows a setting to configure the default values to use for the status field.

**Reply by rijkvanzanten** [reply]
Those recommended defaults are hardcoded in the codebase at the moment. There's no configuration to override those. 

That being said, you _can_ "Duplicate" a field from one collection to another. I think the best in between approach would be to setup the dropdown field manually, and then copying it to the other collections where you wanna have it the same way 🙂

**Reply by benhaynes** [reply]
You should also be able to click the field label to copy the raw status values more easily between collections/fields.

**Reply by log4shell** [reply]
Thanks for the suggestions, I guess I'll have to copy the field setup for each new collection for now. I still think it would be a good feature to add the ability to customize these values from the admin settings in the future if you get a chance to work on it.
